src/admin-events/agent.js

```diff
@@ -111,21 +116,111 @@ function createAlert(
   return Finding.fromObject(finding);
 }
 
-const initialize = async (config) => {
+const validateConfig = (config, abiOverride = null) => {
+  let ok = false;
+  let errMsg = "";
+
+  if (!isFilledString(config.developerAbbreviation)) {
```

Comment:
The approach used by this validation function should be applied to other bots that don't have it (`account-balance` is the first one that comes to mind).